docs/source/md/cli_references.md

```diff
@@ -0,0 +1,63 @@
+## Command Line Interface
+
+Aim CLI offers a simple interface to easily organize and record your experiments.
+Paired with the [Python Library](#sdk-specifications), Aim is a powerful utility to record, search and compare AI experiments.
+Here are the set of commands supported:
+
+
+| Command       | Description                                                          |
+| --------------| -------------------------------------------------------------------- |
+| `init`        | Initialize the `aim` repository.                                     |
+| `version`     | Displays the version of aim cli currently installed.                 |
+| `experiment`  | Creates a new experiment to group similar training runs into.        |
+| `up`          | Runs Aim web UI for the given repo                                   |
+
+### init
+__**This step is optional.**__
+Initialize the aim repo to record the experiments.
+```shell
+$ aim init
+```
+Creates `.aim` directory to save the recorded experiments to.
+Running `aim init` in an existing repository will prompt the user for re-initialization.
+
+  **_Beware:_** Re-initialization of the repo clears `.aim` folder from previously saved data and initializes new repo.
+  **_Note:_** This command is not necessary to be able to get started with Aim as aim is automatically initializes with the first aim function call.
+
+### version
+Display the Aim version installed.
+```shell
+$ aim version
+```
+
+### experiment
```

Comment:
same here; we don't have `aim experiment` anymore

Reply:
thnks, will fix this.

Reply:
let's do another iteration and go through this afterwards ?

Reply:
resolving since we have a task for CLI section